Sources/Clickstream/Dependency/Clickstream/Analytics/EventRecorder.swift

```diff
@@ -133,6 +144,15 @@ class EventRecorder: AnalyticsEventRecording {
         }
         return BatchEvent(eventsJson: eventsJson, eventCount: eventCount, lastEventId: lastEventId)
     }
+
+    func logEventPrettier(event: ClickstreamEvent) {
+        log.debug("Event saved, event name: \(event.eventType)")
+        print("attributes: {")
```

Comment:
Why using print here

Reply:
`log.debug()` function will log the default time and class path information, when the attribute value length is long, for example event `product_exposure`, the log will looks unfriendly:
<img width="857" alt="image" src="https://github.com/awslabs/clickstream-swift/assets/33129495/34ddd0ac-6598-4b42-b129-3452bc6580cf">
when use `print(xxx)`can directly print the content in output：
<img width="999" alt="image" src="https://github.com/awslabs/clickstream-swift/assets/33129495/135539ff-4d0d-49d6-9ed4-bb8566388f57">
Analytics SDK event logs are generally printed as above.

Reply:
Using the system's print directly will result in the loss of the capability to overwrite the configuration of the logger, which is not something I favor.

Reply:
I also think it's not a good way. Fixed it for using common log util and also have prettier output.